sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/MetricQueryResults.java

```diff
@@ -18,24 +18,68 @@
 package org.apache.beam.sdk.metrics;
 
 import com.google.auto.value.AutoValue;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.List;
 import org.apache.beam.sdk.annotations.Experimental;
 import org.apache.beam.sdk.annotations.Experimental.Kind;
+import org.apache.beam.vendor.grpc.v1p13p1.com.google.common.collect.ImmutableList;
 
 /**
  * The results of a query for metrics. Allows accessing all of the metrics that matched the filter.
  */
 @AutoValue
 @Experimental(Kind.METRICS)
 public abstract class MetricQueryResults {
-  /** Return the metric results for the getCounters that matched the filter. */
+  /** Return the metric results for the counters that matched the filter. */
   public abstract Iterable<MetricResult<Long>> getCounters();
 
-  /** Return the metric results for the getDistributions that matched the filter. */
+  /** Return the metric results for the distributions that matched the filter. */
   public abstract Iterable<MetricResult<DistributionResult>> getDistributions();
 
-  /** Return the metric results for the getGauges that matched the filter. */
+  /** Return the metric results for the gauges that matched the filter. */
   public abstract Iterable<MetricResult<GaugeResult>> getGauges();
 
+  static <T> void printMetrics(String type, Iterable<MetricResult<T>> metrics, PrintStream ps) {
```

Comment:
`StringBuilder` instead of `PrintStream`

Reply:
done